### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by reshmagiliyal**
Modified the net_demo_installer and setup_k8_cluster.sh script to give contiv network version through CLI if needed

### CODE REVIEW

File: k8s/setup_k8s_cluster.sh [hunk 1]
```diff
@@ -60,7 +60,8 @@ username=$1
 : ${k8sVer:=v1.1.4}
 
 # contiv version
-: ${contivVer:=v0.1-05-19-2016.08-34-56.UTC}
+: ${contivVer:=$v0.1-05-19-2016.08-34-56.UTC}
```
Comment (jojimt) [inline k8s/setup_k8s_cluster.sh:63]: Please undo this change. No need for a $ before v0...

File: net/net_demo_installer [hunk 2]
```diff
@@ -145,8 +145,12 @@ function sudoExec() {
     sudo -E $@
 }
 
-contiv_network_version="v0.1-05-19-2016.08-34-56.UTC"
+: ${contiv_network_version:=v0.1-05-19-2016.08-34-56.UTC}
+echo "Using version: $contiv_network_version"
+
 docker_version="1.11.1"
+
+#docker_version="1.11.1"
```
Comment (jainvipin) [inline net/net_demo_installer:153]: why comment out 1.11.1 version? do you want to change something else to default?
